### PR TITLE
test: add request bodies used in tests

### PR DESCRIPTION
**Pull request by athith-g**
Adds the proTES request bodies that will be used in the middleware tests.

### CODE REVIEW

File: tests/tasks/task_bodies.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
+from pathlib import Path
+
+DIR = Path(__file__).parents[2]
+input_dir = DIR / "inputs"
+output_dir = DIR / "outputs"
+
+uppercase_task_body = {
```
[the remark this answers is not in the document]
Comment (uniqueg) [reply tests/tasks/task_bodies.py:7]: Good comment, but doesn't necessarily need to be done in this PR - up to you.

**Review by uniqueg — APPROVED** [review]
Look at the one open Sourcery AI comment. If you add additional requests later, then close the conversation and merge. Otherwise, add additional requests and ask for a re-review.

Also, before I forget to mention that (goes for all PRs): make sure you merge PRs in the right order for them to make sense. And, of course, after merging one PR, make sure to merge the updated `main` branch into the feature branch of the PR that you plan to merge next.

**Comment by athith-g** [comment]
That makes sense I think I it seems most logical to merge the input files before merging the task bodies and tests.

**Comment by athith-g** [comment]
@uniqueg Sorry about the successive review requests- I should've merged the other branch into this one first.

**Comment by uniqueg** [comment]
> @uniqueg Sorry about the successive review requests- I should've merged the other branch into this one first.

No worries :)

**Review by uniqueg — APPROVED** [review]

**Pull request merged**